Source/WebCore/page/Page.cpp

```diff
@@ -1675,6 +1683,15 @@ void Page::updateRendering()
 
     m_lastRenderingUpdateTimestamp = MonotonicTime::now();
 
+#if HAVE(WORK_INTERVAL_API)
+    static std::once_flag onceFlag;
```

Comment:
This code needs to move somewhere else. The `WorkgroupIntervalScope` should be a one-liner here.

Reply:
Thanks for reviewing! I will address these issues.